rainbow-me/rainbowkit
# chore: track web3modal state in `useConnectModal` hook

## Changes
- `connectModalOpen` state in `useConnectModal` hook now includes the official WalletConnect modal open status

## What to test
1. Go to [this](https://rainbowkit-example-843jmapj4-rainbowdotme.vercel.app/) preview link
2. Make sure whenever you open the official WalletConnect modal the button text should switch from `Open connect modal` -> `Connect modal opened` or vice versa if it's closed
![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/8283a3ea-af3e-4eb8-8f7f-c5e1afa2e93c)
